plugins/inputs/file/file.go

```diff
@@ -12,6 +13,7 @@ import (
 
 type File struct {
 	Files  []string `toml:"files"`
+	File   string   `toml:"file_tag"`
```

Comment:
Call this field `FileTag`, this just reduces the cognitive load keeping track of what option a variable maps to.  Also, I feel like we need the tag part to properly describe what the option does.

Reply:
I was thinking something would be better for the name.  Maybe `input_file_tag_name`?  Anything without tag sounds like it's overriding the file name in some way.

Reply:
I think we should refer to it as the `file` instead of `input_file` because we already named the `files` option without the input prefix.  The `_name` suffix would be reasonable, though we do have quite a few pre-existing variables that are similar, and I'd like to keep them in the same style unless we the new suffix is enough of an improvement to justify a switch.

In `kafka_consumer` and `mqtt_consumer` we have `topic_tag = ""`, which works the same, we also have `database_tag` in `http_listener` There is also, very soon to be merged, a pull request for the `prometheus` input with a `url_tag = ""`.

Reply:
So I would stick with `file_tag` as the option name for those reasons.